bin/generate

```diff
@@ -227,7 +227,7 @@ var autoregistrationOutput = [
     "let maxDependencies = \(dependenciesCount)",
     "let maxArguments = \(argumentsCount)",
     "\n",
-    "public extension Container {\n",
+    "extension Container {\n",
```

Comment:
I saw this was removed in #45 , does it make sense to remove it in this PR as well?

Reply:
Looks like we need to keep this, build is failing.